tests/tests_ntp_provider6.yml

```diff
@@ -0,0 +1,72 @@
+
+- name: Configure NTP with OS release non-default provider and then change it to the default provider
+  hosts: all
+  vars:
+    is_dist: "{{ ansible_distribution in ['RedHat', 'CentOS'] }}"
+    is_vers: "{{ ansible_distribution_version is version_compare('7.0', '<') }}"
```

Comment:
AFAICS there could be a single `is_el6` variable that would make it easier to understand:
```
is_el6: "ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_version is version('7.0', '<')
```

Also `version_compare` is now deprecated. Another improvement would be to use `ansible_distribution_major_version` and compare it to `'6'`.